notify/impl.go

```diff
@@ -365,23 +366,47 @@ func (n *Email) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
 		fmt.Fprintf(wc, "%s: %s\r\n", header, mime.QEncoding.Encode("utf-8", value))
 	}
 
-	fmt.Fprintf(wc, "Content-Type: text/html; charset=UTF-8\r\n")
+	buffer := &bytes.Buffer{}
+	multipartWriter := multipart.NewWriter(buffer)
+
 	fmt.Fprintf(wc, "Date: %s\r\n", time.Now().Format(time.RFC1123Z))
+	fmt.Fprintf(wc, "Content-Type: multipart/alternative;  boundary=\"%s\"\r\n", multipartWriter.Boundary())
+
 
 	// TODO: Add some useful headers here, such as URL of the alertmanager
 	// and active/resolved.
 	fmt.Fprintf(wc, "\r\n")
 
-	// TODO(fabxc): do a multipart write that considers the plain template.
+	// Html template
+	multipartWriter.CreatePart(textproto.MIMEHeader{"Content-Type": {"text/html; charset=UTF-8"}})
```

Comment:
According to the documentation of `CreatePart` the `io.Writer` it returns should be used to write the output of the template into.
Same below when creating the part for the text template.

https://godoc.org/mime/multipart#Writer.CreatePart

The returned error should be checked as well.